src/usdb_syncer/constants.py

```diff
@@ -164,3 +164,43 @@ class Usdb:
     "Yoruba": "yor",
     "Zulu": "zul",
 }
+
+QUOTATION_MARKS = {
+    # See: https://en.wikipedia.org/wiki/Quotation_mark#Summary_table
+    ("Bosnian", "Finnish", "Swedish"): ("”", "”"),
+    ("Breton", "French"): ("« ", " »"),
+    ("Croatian", "Dutch", "Hebrew", "Hungarian", "Polish", "Romanian"): ("„", "”"),
+    (
+        "Catalan",
+        "Galician",
+        "Greek",
+        "Greek (romanized)",
+        "Italian",
+        "Norwegian",
+        "Portuguese",
+        "Russian",
+        "Russian (romanized)",
+        "Spanish",
+        "Ukrainian",
+    ): ("«", "»"),
+    (
+        "Chinese (romanized)",
+        "English",
+        "Hindi",
+        "Hindi (romanized)",
+        "Indonesian",
+        "Irish",
+        "Korean",
+        "Korean (romanized)",
+        "Portuguese (Brazil)",
+        "Scots",
+        "Turkish",
+        "Vietnamese",
+    ): ("“", "”"),
+    ("Czech", "Estonian", "Icelandic", "Serbian", "Slovak", "Slovenian"): ("„", "“"),
+    ("Danish", "German"): ("»", "«"),
+    ("Japanese", "Japanese (romanized)"): ("「", "」"),
+    ("Welsh",): ("‘", "’"),
+}
+
+QUOTATION_MARKS_TO_REPLACE = {'"', "„", "”", "«", "»", "“", "「", "」"}
```

Comment:
Is this the subset of all quotation marks above minus the ones that could also be an apostrophe?

Reply:
Plus the typewriter (upright) quotation marks " (first entry).